internal/commands/exec/command.go

```diff
@@ -15,15 +15,17 @@ import (
 	"sync"
 	"time"
 
-	"github.com/hashicorp/consul-api-gateway/internal/consul"
 	"github.com/hashicorp/consul-server-connection-manager/discovery"
 
+	"github.com/hashicorp/consul-api-gateway/internal/consul"
```

Comment:
Should be further down w/ the other local pkg

Reply:
oh weird, lemme check out our implementation of `make fmt` to make sure it's not missing anything